autoload/jetpack.vim

```diff
@@ -288,20 +285,12 @@ function! s:display() abort
 
   let line_count = 1
   for pkg in s:pkgs
-    call s:setbufline(line_count, printf('%s %s', msg[pkg.progress.type], pkg.name))
-    let line_count += 1
-
     let output = pkg.progress.output
     let output = substitute(output, '\r\n\|\r', '\n', 'g')
+    let output = substitute(output, '^From.\{-}\zs\n\s*', '/compare/', '')
 
-    if pkg.progress.type ==# s:progress_type.update
-      let from_to = matchstr(output, 'Updating\s*\zs[^\n]\+')
-      if from_to !=# ''
-        call s:setbufline(line_count, printf('  Changes %s/compare/%s', pkg.url, from_to))
-        let line_count += 1
-      endif
-    endif
-
+    call s:setbufline(line_count, printf('%s %s', msg[pkg.progress.type], pkg.name))
+    let line_count += 1
```

Comment:
I would like to hear the summary of this simplification.

Reply:
The previous implementation has a problem that might occur if the repo will rebased. (The `Updating...` message is omitted in such case in my environment.)


It seems the `From %URL% %FROM%...%TO%` is always displayed so I changed to use this message here. (If we hope to support it perfectly, we should use `git log` command for displaying the change history.)

Reply:
It's reasonable. Thank you for the explanation.